lib/ansible/modules/storage/netapp/na_ontap_net_vlan.py

```diff
@@ -81,8 +75,6 @@ def __init__(self):
             parent_interface=dict(required=True, type='str'),
             vlanid=dict(required=True, type='str'),
             node=dict(required=True, type='str'),
-            interface_name=dict(required=False, type='str'),
```

Comment:
`interface_name` has been removed which was in using in 2.6, though docs don't explain what you should use instead

Reply:
Where should we document this?

ONTAP required interface name to be <parent-interface>-<vlanid> anything else would result in an error. Since we are already getting these 2 part of we don't need to ask the user for the interface name to create the correct name.

Reply:
In the `DOCUMENTION` section, possibly under `notes:`

Also if users of this module in Ansible 2.6 will need to modify there playbooks to work with Ansible 2.7 is should also be documented in docs/docsite/rst/porting_guides/porting_guide_2.7.rst